### PR TITLE
Add mobile SDK MeetingSessionStatusCode handling to FAQ

### PR DESCRIPTION
**Pull request by georgezy-amzn**
**Issue #:**

**Description of changes:**

Added 3 topics to FAQ related to mobile MeetingSessionStatusCode:
 - What does each MeetingSessionStatusCode mean?
 - How should I handle each MeetingSessionStatusCode in my application?
 - How to determine if the meeting session stop will be self-recoverable or terminal?

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hensmi-amazon — APPROVED** [review]

**Review by linsang21 — APPROVED** [review]

File: guides/07_FAQs.md [hunk 1]
```diff
@@ -78,6 +78,11 @@
     - [How can I speed up my meeting join times?](#how-can-i-speed-up-my-meeting-join-times)
   - [Messaging](#messaging)
     - [How do I receive Amazon Chime SDK channel messages without using the Amazon Chime SDK for JavaScript?](#how-do-i-receive-amazon-chime-sdk-channel-messages-without-using-the-amazon-chime-sdk-for-javascript)
+  - [Mobile SDK](#mobile-sdk)
+    - [What does each MeetingSessionStatusCode mean?](#what-does-each-meetingsessionstatuscode-mean)
```
Comment (devalevenkatesh) [inline guides/07_FAQs.md:82]: nit: Lets backtick the coding naming constructs. For example: `MeetingSessionStatusCode` here. I see you are using it in some places but suggest to do this overall for consistency.

File: guides/07_FAQs.md [hunk 2]
```diff
@@ -78,6 +78,11 @@
...
+    - [How should I handle each MeetingSessionStatusCode in my application?](#how-should-i-handle-each-meetingsessionstatuscode-in-my-application)
+    - [How to determine if the meeting session stop will be self-recoverable or terminal?](#how-to-determine-if-the-meeting-session-stop-will-be-self-recoverable-or-terminal)
```
Comment (devalevenkatesh) [inline guides/07_FAQs.md:84]: nit: the -> a meeting session

File: guides/07_FAQs.md [hunk 3]
```diff
@@ -620,4 +625,40 @@ Meeting join comprises several steps, one of which is establishing a signaling c
...
+
+These status codes can be used for logging, debugging, and notification to end users, but in most cases should not be used for any retry behavior, as Chime SDK will already be retrying non-terminal errors. The table below illustrates how each code should be handled.
+
+| MeetingSessionStatusCode | Callbacks | How to handle |
+| -------- | ------- | ------- |
+| OK(iOS/Android) | `audioSessionDidStopWithStatus(MeetingSessionStatus)`  `videoSessionDidStartWithStatus(MeetingSessionStatus)`  `videoSessionDidStopWithStatus(MeetingSessionStatus)` | This code indicates the audio/video sessions on the device have been started/stopped without any errors. When received from AudioVideoObserver callbacks, your application may want to notify your user about the success of session start/stop.
```
Comment (devalevenkatesh) [inline guides/07_FAQs.md:643]: - We can probably call out why specific codes are present in Android vs iOS SDK for example the `Left` one for builder clarification.
- Probably you may have done so, but, suggest checking the reasoning once with the JS SDK statuses as well and if we need any changes for consistency.

**Comment by devalevenkatesh** [comment]
You need to run the `npm run build:release` to update and generate the HTML docs as well to get the "Continuous Integration Workflow / Build and Run Unit Tests" passing.

File: guides/07_FAQs.md (hunk 3 above)
Comment (georgezy-amzn) [reply guides/07_FAQs.md:643]: I don't have enough info to tell why there are discrepancies between iOS and Android, we just want to document what we have at this moment. There is a backlog to investigate and improve those codes.

**Review by ltrung — APPROVED** [review]

**Pull request merged**